lib/zendesk_apps_tools/server.rb

```diff
@@ -4,7 +4,6 @@
 module ZendeskAppsTools
   class Server < Sinatra::Base
     set :protection, :except => :frame_options
-    set :public_folder, proc { "#{settings.root}/assets" }
```

Comment:
there's probably some magic you can do to mimic this.

before_public_folder_call.method_send(:access_control_allow_origin) or something, but I don't know magic rails.